services/QuillLMS/app/services/vitally_integration/serialize_vitally_sales_organization.rb

```diff
@@ -207,11 +209,19 @@ def activities_finished_query(district)
         .where('activity_sessions.state = ?', 'finished')
     end
 
+    private def first_subscription
```

Comment:
Again, this is just style, and we don't have a requirement to use the shorter style, I just like it:
```ruby
private def first_subscription = @district.subscriptions.order(start_date: :asc).first
private def earliest_premium_start_date = first_subscription&.start_date || VITALLY_NOT_APPLICABLE
private def current_premium_start_date = @district.subscription&.start_date || VITALLY_NOT_APPLICABLE
private def local_premium_months = @district.subscriptions.map(&:length_in_months).sum
```